api-registration/server/index.js

```diff
@@ -0,0 +1,70 @@
+require('dotenv/config');
+const pg = require('pg');
+const argon2 = require('argon2'); // eslint-disable-line
+const express = require('express');
+const ClientError = require('./client-error');
+const errorMiddleware = require('./error-middleware');
+
+const db = new pg.Pool({ // eslint-disable-line
+  connectionString: process.env.DATABASE_URL,
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
+
+const app = express();
+
+const jsonMiddleware = express.json();
+
+app.use(jsonMiddleware);
+
+app.post('/api/auth/sign-up', (req, res, next) => {
+  const { username, password } = req.body;
+  if (!username || !password) {
+    throw new ClientError(400, 'username and password are required fields');
+  }
+
+  /* your code starts here */
+
+  argon2.hash(password)
+    .then(hashedPW => {
+      const sql = `INSERT INTO "users"
+      (
+        "username",
+        "hashedPassword"
+      )
+      VALUES
+      (
+        $1,
+        $2
+      )
+      RETURNING *`;
```

Comment:
We don't want to send back the `hashedPassword` to the client. Here you can update your `returning` statement to only return the columns you want to send to the client.